components/Layout.vue

```diff
@@ -22,10 +22,10 @@
       </article>
     </div>
     <nav
-      class="prose-a:no-underline sticky top-[4.5rem] flex flex-col w-full px-4 bg-white drop-shadow-sm lg:max-h-page lg:self-start lg:bg-transparent lg:backdrop-blur-none lg:flex-none lg:overflow-y-auto lg:py-16 lg:col-span-1"
+      class="prose-a:no-underline sticky top-[4.5rem] flex flex-col w-full bg-white drop-shadow-sm lg:max-h-page lg:self-start lg:bg-transparent lg:backdrop-blur-none lg:flex-none lg:overflow-y-auto lg:py-16 lg:col-span-1"
     >
       <button
-        class="px-0 py-3 gap-1 flex items-center focus:outline-none focus:ring-0 mb-0 lg:hidden"
+        class="type-overline text-brand-accent px-6 py-3 gap-1 flex items-center focus:outline-none focus:ring-0 mb-0 lg:hidden"
```

Comment:
Any reason why this has been given `px-6`? It doesn't align with the headings now

![image](https://user-images.githubusercontent.com/70119888/211673664-53777056-d00d-4416-9bae-da54d7fd20ef.png)

Reply:
Thanks @AndyClifford that was an incorrect addition, have now removed this.